### PR TITLE
[breaking] drop Node 14 support

### PR DESCRIPTION
**Pull request by benmccann**
closes https://github.com/sveltejs/kit/issues/4916

### CODE REVIEW

File: packages/adapter-vercel/index.js [hunk 1]
```diff
@@ -395,8 +395,8 @@ function get_node_version() {
 	const full = process.version.slice(1); // 'v16.5.0' --> '16.5.0'
 	const major = parseInt(full.split('.')[0]); // '16.5.0' --> 16
 
-	if (major < 14) {
-		throw new Error(`SvelteKit only support Node.js version 14 or greater (currently using v${full}). Consult the documentation: https://vercel.com/docs/runtimes#official-runtimes/node-js/node-js-version`)
+	if (major < 16) {
+		throw new Error(`SvelteKit only support Node.js version 16 or greater (currently using v${full}). Consult the documentation: https://vercel.com/docs/runtimes#official-runtimes/node-js/node-js-version`)
```
Comment (mrkishi) [inline packages/adapter-vercel/index.js:399]:
```suggestion
		throw new Error(`SvelteKit only supports Node.js version 16 or greater (currently using v${full}). Consult the documentation: https://vercel.com/docs/runtimes#official-runtimes/node-js/node-js-version`)
```

**Comment by Rich-Harris** [comment]
AFK or I'd do this myself right now - we should get rid of the node 14 crypto stuff as well

**Comment by Rich-Harris** [comment]
Actually, scratch that. We should change install-fetch to be install-polyfills or something and assign `require('node: crypto').webcrypto` to `globalThis.crypto`, and put any other things we need in there (like `AbortController` and `ReadableStream` etc). Will do that in a follow-up

**Review by Rich-Harris — APPROVED** [review]

**Comment by Cinci7** [comment]
Your not done yet should of read the whole lesson plan when you had the chance.

**Pull request merged**